docker-compose-dev.yml

```diff
@@ -0,0 +1,18 @@
+db:
+  image: mysql:5.7
+  ports:
+    - "3306:3306"
+  environment:
+    MYSQL_DATABASE: ralph_ng
+    MYSQL_ROOT_PASSWORD: ralph_ng
+    MYSQL_USER: ralph_ng
+    MYSQL_PASSWORD: ralph_ng
+  volumes:
+    - ./volumes/static:/opt/static
```

Comment:
do we really need to map this explicit to host filesystem? Why not just `- /opt/static`?